src/components/Comment/Comments.scss

```diff
@@ -0,0 +1,74 @@
+@import "~styles/main";
+
+.comment {
+  @include font(Poppins);
+  margin: 20px auto;
+  width: 60%;
+
+  //shame: this is very hacky method. If found an alternative, please replace
+  .avatar {
+    position: fixed;
+    margin: 0px -60px;
+  }
+
+  &__top {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 0 0 5px 0;
+    padding: 0;
+    .comment__author {
+      font-weight: 700;
+      font-size: 0.75rem;
+      display: inline-block;
+      vertical-align: middle;
+      // margin-right: 200px;   // shame- cant give margin right of text
+    }
+    em {
+      font-weight: 400;
+      font-size: 0.66rem;
+      display: block;
+      color: lighten(color(pure-black), 50%);
+    }
+    &__detailsIcon {
+      float: right;
+    }
+  }
+
+  &__mid {
+    font-size: 0.66rem;
+    font-weight: 400;
+    color: lighten(color(pure-black), 20%);
+  }
+
+  &__bot {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    font-size: 16px;
+    iconContainer {
+      display: grid;
+      justify-content: center;
+    }
+    .icon {
+      vertical-align: middle;
+    }
+    .author {
+      vertical-align: middle;
+      display: inline-block;
+    }
+    span {
+      box-sizing: border-box;
+      margin: 5px;
+      line-height: 24px;
+    }
+    &__name {
+      font-size: 0.666rem;
+      display: inline-block;
+      color: lighten(color(pure-black), 40%);
+    }
+    .reply {
+      font-size: 0.85rem;
```

Comment:
make your units consistent, don't do haphazardly. either use `PX` or `REM` not both

Reply:
Using rem or em for fonts is ok but not recommended for setting margins, padding
https://stackoverflow.com/questions/45822322/when-to-choose-rem-over-px#:~:text=You%20can%20use,are%20mostly%20used.
If there is specific reason to use rem for margin, I could change

Reply:
You still need more research for a specific reason (Y)

[Read this - 1](https://graphicdesign.stackexchange.com/questions/2147/which-unit-should-be-used-in-web-design-to-define-space-margin-padding-etc)

[Read this - 2](https://daveredfern.com/use-units-css/)

[Read this - 3](https://gist.github.com/basham/2175a16ab7c60ce8e001)

[Read this - 4](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units)

[Read this - 5](https://medium.com/swlh/css-units-which-ones-to-use-and-avoid-31e4ed461f9)

[Read this - 6](https://www.digitalocean.com/community/tutorials/how-to-use-common-units-in-css)

[Read this - 7](https://www.digitalocean.com/community/tutorials/css-css-units-explained)

[Read this - 8](https://labs.tadigital.com/index.php/2020/06/18/common-units-in-css-and-when-to-use/)

[Read this - 9](https://getflywheel.com/layout/choose-css-unit-create-better-site-layouts-how-to/)

[Read this - 10](https://daveredfern.com/use-units-css/)

[Read this - 11](https://css-tricks.com/confused-rem-em/)


Hope you get the answer, And please share with us, what should we need to use :)

Reply:
You still need more research for a specific reason (Y)

[Read this - 1](https://graphicdesign.stackexchange.com/questions/2147/which-unit-should-be-used-in-web-design-to-define-space-margin-padding-etc)

[Read this - 2](https://daveredfern.com/use-units-css/)

[Read this - 3](https://gist.github.com/basham/2175a16ab7c60ce8e001)

[Read this - 4](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units)

[Read this - 5](https://medium.com/swlh/css-units-which-ones-to-use-and-avoid-31e4ed461f9)

[Read this - 6](https://www.digitalocean.com/community/tutorials/how-to-use-common-units-in-css)

[Read this - 7](https://www.digitalocean.com/community/tutorials/css-css-units-explained)

[Read this - 8](https://labs.tadigital.com/index.php/2020/06/18/common-units-in-css-and-when-to-use/)

[Read this - 9](https://getflywheel.com/layout/choose-css-unit-create-better-site-layouts-how-to/)

[Read this - 10](https://daveredfern.com/use-units-css/)

[Read this - 11](https://css-tricks.com/confused-rem-em/)

and do check out what these popular frameworks use,

[bootstrap](https://getbootstrap.com/docs/5.0/utilities/spacing/)

[Foundation](https://get.foundation/sites/docs/global.html)

[TailwindCSS Margin](https://tailwindcss.com/docs/margin)

[TailwindCSS Padding](https://tailwindcss.com/docs/padding)

[Styled Component](https://styled-components.com/docs/advanced)


Hope you get the answer. And **please recommend**, what we should use :)